docker_templates/templates/snippet/setup_ros_sources.Dockerfile.em

```diff
@@ -26,12 +29,28 @@ else:
             apt_suffix += '-testing'
             source_suffix = 'testing'
     repo_url = f'http://packages.ros.org/ros{apt_suffix}/ubuntu'
+
+# Get the latest tag
```

Comment:
this should be in the else block as `apt_suffix` variable doesnt exist in the if branch